api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -56,10 +56,16 @@ type VRGAsyncSpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern=`^\d+[mhd]$`
 	SchedulingInterval string `json:"schedulingInterval"`
+
+	//+optional
+	PeerClasses []PeerClass `json:"peerClasses,omitempty"`
 }
 
 // VRGSyncSpec has the parameters associated with MetroDR
-type VRGSyncSpec struct{}
+type VRGSyncSpec struct {
```

Comment:
Left a comment at the design [spec](https://github.com/RamenDR/ramen/pull/1403/files#r1662758687) to see if we need these separated. Leaving it as is for now.

Reply:
Above is resolved, can retain as is.